tests/tests/swfs/avm1/movieclip_begin_gradient_fill/test.toml

```diff
@@ -0,0 +1,8 @@
+ignore = true
```

Comment:
Ignored because there's still more unimplemented, or a mistake?

Reply:
We don't exactly match Flash here and this causes the number of outliers to be very high.

For instance, here's what the `difference-color-windows-Vulkan.png` file looks like when running this test (`avm1/movieclip_begin_gradient_fill`):

![new](https://github.com/ruffle-rs/ruffle/assets/14274782/3a57f202-d732-446f-87b0-32af22590fc4)

For the record, this is the same thing in master, with a few extra issues:
![old](https://github.com/ruffle-rs/ruffle/assets/14274782/dcda1eb1-653b-422b-9e31-aeb036701860)

Reply:
Ah, did you make expected.png from flash?

Reply:
Yes, this is made based on the Flash projector.

Reply:
Was flash set to low quality?

If it visually looks the same, use the expected.png that ruffle generates (it's just actual.png), it's to stop regressions more than to make sure it's identical to flash.

Reply:
No, the quality was set to high. I tested setting it to low, this only affects the way borders around the squares are rounded (no alpha), otherwise everything else looks the same.

Reply:
Understood. Can you please switch to using the image Ruffle generated for `expected.png` and unignoring the test though? The purpose isn't to test against flash (humans should do that), but to test against two versions of Ruffle and make sure we don't regress it.

Reply:
Alright, thanks for clarifying this. I made the changes (and also renamed the `source.txt` files to `source.as` as suggested in the other PR).

Reply:
Looks like we'll need a tolerance of 3 on both tests to account for the test runners, and then we're good, thank you!

Reply:
Thank you for the assistance. 😉